transform.go

```diff
@@ -309,5 +309,47 @@ func transformImage(m image.Image, opt Options) image.Image {
 		m = imaging.FlipH(m)
 	}
 
+	// trim
+	if opt.Trim {
+		m = trimEdges(m)
+	}
+
 	return m
 }
+
+func trimEdges(img image.Image) image.Image {
+	bounds := img.Bounds()
+	minX, minY, maxX, maxY := bounds.Max.X, bounds.Max.Y, bounds.Min.X, bounds.Min.Y
```

Comment:
are you intentionally flipping min and max values here?

Reply:
Yeah, they are only initialized with min and max values, but after the loop minX/minY will be reduced from max to the smallest coordinates of non-matching pixels, and the opposite for maxY/minY (they will be increase from minimum to largest coordinates for non-matching pixels.